modules/light-clients/08-wasm/keeper/keeper_test.go

```diff
@@ -22,14 +28,36 @@ type KeeperTestSuite struct {
 	chainC *ibctesting.TestChain
 }
 
+func init() {
+	ibctesting.DefaultTestingAppInit = setupTestingApp
+}
+
+// setupTestingApp provides the duplicated simapp which is specific to the 08-wasm module on chain creation.
+func setupTestingApp() (ibctesting.TestingApp, map[string]json.RawMessage) {
```

Comment:
these two functions are re-used in every package that defines its own testsuite (keeper_test and types_test so far). we could move them someplace in testing I'd suppose.